### ISSUE TITLE
Table 5 of Appendix A lists lots of now obsoleted curves (draft 13).

### ISSUE THREAD

**Issue by grubba** [tlswg/rfc4492bis#33]
The table should probably be truncated to hold only the last three entries.
Cf Section 5.1.1: NamedCurve.

I guess the ecdh_x* curves could be added.

**Comment by yoavnir** [comment]
I disagree.  Appendix A was placed in RFC 4492 because different curves have different names when different standards bodies (none of which is the IETF) refer to them. Just because we're deprecating some of them does not mean we don't need the table, if only for knowing that secp256k1 is not another name for P-256, but one of the deprecated curves.

As for curve25519 and curve448, these are not referenced by NIST, ANSI, and SECG, so only our names for them exist.  (and "goldilocks")
